svm/Cargo.toml

```diff
@@ -53,15 +53,12 @@ solana-svm-transaction = { workspace = true }
 solana-system-interface = { workspace = true }
 solana-sysvar-id = { workspace = true }
 solana-timings = { workspace = true }
-solana-transaction-context = { workspace = true }
+solana-transaction-context = { workspace = true, features = ["debug-signature"] }
```

Comment:
the dependency should be redeclared with the feature flag in `[dev-dependencies]`, not added to prod `[dependencies]`

Reply:
something else is wrong then. this should never be used in prod

Reply:
@Lichtso @LucasSte is this debug signature stuff dead code? can't find it used otherwise in monorepo or sdk